physical/latency.go

```diff
@@ -0,0 +1,90 @@
+package physical
+
+import (
+	"math/rand"
+	"time"
+
+	log "github.com/mgutz/logxi/v1"
+)
+
+const (
+	// DefaultJitterPercent is used if no cache size is specified for NewCache
+	DefaultJitterPercent = 20
+)
+
+// LatencyInjector is used to add latency into underlying physical requests
+type LatencyInjector struct {
```

Comment:
Since this and (`TransactionalLatencyInjector`) is really an implementation of the `Physical` interface, should we call this `LatencyBackend` to keep things uniform with other physical backend implementations?

Reply:
It is not a backend itself and just wraps some other backend that is why I didn't call it a backend.  It is more analogous to the `Cache` wrapper.